web-frontend/src/pages/TransferCoins/ReceiveCoins/ReceiveCoins.tsx

```diff
@@ -2,15 +2,24 @@ import { Button } from '@massalabs/react-ui-kit';
 import CopyAddress from './CopyAddress';
 import { FiLink } from 'react-icons/fi';
 import Intl from '../../../i18n/i18n';
+import QRCodeReact from 'qrcode.react';
 
 function ReceiveCoins({ ...props }) {
+  const { account } = props;
+  const address = account.address;
+  const formattedAddress = address.slice(0, 4) + '...' + address.slice(-4);
+  const VITE_BASE_APP = import.meta.env.VITE_BASE_APP;
+  const baseURL = window.location.origin;
+
+  const presetURL = `${baseURL}${VITE_BASE_APP}/send-redirect/?to=${address}`;
+  console.log(presetURL);
```

Comment:
!? 🤔

Reply:
aaah sorry haha, it is for testing as the copytext button is not here yet on this pr